packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -180,6 +182,14 @@ class BaseDescopeWc extends HTMLElement {
     return res === 'true';
   }
 
+  get storagePrefix() {
+    return this.getAttribute('storage-prefix') || '';
+  }
+
+  get preview() {
+    return !!this.getAttribute('preview');
+  }
```

Comment:
I'm not sure that those attributes should be exposed as props / inputs of the web-sdk.

maybe we should consider taking them from local storage, so they will not be as a part of the sdk/web-component inputs?

(I'm not sure they both will be needed by customers, and if they do - we can easily expose

@Bars92 @nirgur lmk what you think

Reply:
i prefer keep it straight forward and not invent any more mechanisms to confuse users. Those props can come in handy, and a local storage is far too dangerous to apply and maintain regardless..

Reply:
I don't think this is for users at the moment, this is currently for internal usage (similar to `content .base.url` local storage)

and making the API external will confuse the users more because it will increase the API surface of an already long API options

I don't think local storage is dangerous by any mean

though it will require more work on the caller, but at the moment - this tradeoff is much better - to simplify the SDK that is used by users/customers, and to internalize the complexity

Reply:
The prefix is a good client side solution for multiple flows, if it will be saved in local storage it has to be prefixed with the project and flow, and cleaned whenever used to avoid effecting other flows.. this is way more hard to use and explain, also this will require us to break changes to convert to a prop because the prop is strict by nature. 

Btw the base content local storage was a bad idea that was hard to maintain, this is why it’s mostly automated now

Reply:
we really need a resolution here. @nirgur your opinion is highly desired

Reply:
Maybe preview is for internal usage but prefix (by project ID) is something we already bumped into in the console-app ourself, I think this is something we should expose or have project ID as the default prefix